src/pallavi/BankAccountInfo.java

```diff
@@ -0,0 +1,71 @@
+package pallavi;
+
+public class BankAccountInfo {
+	int totalBalance;
+	String username;
+	int creditIndCount;
+	int debitIndCount;
+	int printIndCount;
+	static int creditCount = 0;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value. and this balance should not hard coded.